R/fimsfit.R

```diff
@@ -418,6 +528,8 @@ FIMSFit <- function(
     number_of_parameters = number_of_parameters,
     timing = timing,
     version = version
+    # json_derived_quantities = json_derived_quantities,
+    # json_density_component = json_density_component
```

Comment:
I am assuming these lines can be removed?

Reply:
Yes. These lines have been removed!